src/cleanlab_codex/project.py

```diff
@@ -145,36 +150,58 @@ def create_access_key(
-            custom_metadata (object, optional): Custom metadata to log in Codex for the query.
-            eval_scores (Dict[str, float], optional): Optional scores to use for the query. When provided, Codex will skip running TrustworthyRAG evaluations on the query and use the provided scores instead.
-            eval_thresholds (Dict[str, float], optional): Optional thresholds to use for evaluating the query. We recommend configuring thresholds on the Project instead and using the same thresholds for all queries.
+            messages (list[ChatCompletionMessageParam]): The full message history from the AI conversation, formatted for OpenAI-style chat completion.
+                This must include the final user message that triggered the AI response. All other arguments—`query`, `context`, and `response`—
+                must correspond specifically to this final user message.
+            response (ChatCompletion | str): The AI-generated response that immediately follows the last user message in `messages`. This is the response being evaluated.
+            context (str): The retrieved context (e.g., from your RAG system) that was supplied to the AI when generating the `response` to the final user query in `messages`.
+            query (str): The original user query that prompted the `response`. This must be the query included in the final user message in `messages`.
+            rewritten_question (str, optional): An optional reformulation of the `query` to make it more self-contained to improve retrieval quality. If this is provided, the query re-writting step is skipped, and the provided `rewritten_question` is used instead.
+            custom_metadata (object, optional): Arbitrary metadata to associate with this validation for logging or analysis inside the Codex project.
+            eval_scores (dict[str, float], optional): Precomputed evaluation scores to bypass automatic scoring.
 
         Returns:
-            ProjectValidateResponse: The response from the validation.
+            ProjectValidateResponse: A structured object with the following fields:
+
+                - should_guardrail (bool): True if the AI system should suppress or modify the response before returning it to the user. When True, the response is considered problematic and may require further review or modification.
+                - escalated_to_sme (bool): True if the query should be escalated to Codex for SME review. When True, the query is logged and may be answered by an expert.
+                - eval_scores (dict[str, ThresholdedEvalScore]): Evaluation scores for different response attributes (e.g., helpfulness, groundedness). Each includes a numeric score and a `failed` flag indicating whether the score falls below threshold.
+                - expert_answer (str | None): If the response was flagged and Codex has a matching prior SME answer, this field contains that expert answer. Otherwise, it is None.
```

Comment:
```suggestion
                - expert_answer (str | None): If it was auto-determined that this query should be escalated to SME, and a prior SME answer for a similar query was found, then this will return that expert answer.  Otherwise, it is None.  
                When available, consider swapping your AI response with the expert answer before serving the response to your user.
```